nucypher/cli/actions.py

```diff
@@ -88,7 +88,8 @@ def get_client_password(checksum_address: str, envvar: str = '') -> str:
 
 
 def get_nucypher_password(confirm: bool = False, envvar="NUCYPHER_KEYRING_PASSWORD") -> str:
-    prompt = "Enter NuCypher keyring password"
+    from nucypher.config.keyring import NucypherKeyring
+    prompt = f"Enter NuCypher keyring password (min. {NucypherKeyring.MINIMUM_PASSWORD_LENGTH} chars.)"
```

Comment:
👍 -  good to give the length for UX - so that the user doesn't enter a password and then it fails, and only then they realize that the password needs to be >= 16 chars.